decoder/DecoderApi_unittest.cpp

```diff
@@ -50,6 +50,11 @@ static FrameData g_h264data[] = {
     g_avc8x8B,
     g_avc8x16,
     g_avc16x16,
+    g_avc8x8I,
+    g_avc8x8P,
```

Comment:
Hi @LindaYu17 
Do not change IBP pattern, it will make wrong deocde image, you can dupplicate he original pattern.
IPBIIIPBIPB etc.